### ISSUE TITLE
NonEmptyAtclauseDescription does not work for @return

### ISSUE THREAD

**Issue by muntyan** [checkstyle/checkstyle#8889]
Check documentation: https://checkstyle.sourceforge.io/config_javadoc.html#NonEmptyAtclauseDescription

```
D:\dev\checkstyle>javac Sample.java

D:\dev\checkstyle>cat Sample.java
class Sample {
    // Bug: `@return` does not produce an error.

    /**
     * @param a xxx
     * @return
     */
    int foo(int a) {
        return a;
    }
}

D:\dev\checkstyle>cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
    "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
    "https://checkstyle.org/dtds/configuration_1_3.dtd">

<module name="Checker">
    <module name="TreeWalker">                                                                                                  
        <module name="NonEmptyAtclauseDescription"/>
    </module>
</module>                                                                                                               
D:\dev\checkstyle>java -Duser.language=en -Duser.country=US -jar checkstyle-8.36.2-all.jar -c config.xml Sample.java
Starting audit...
Audit done.

D:\dev\checkstyle>
```

---------------

Problem: empty `@return` clause is not reported as an error. In the test file above, if you change `@param a xxx` to `@param a`, then it reports an error there, as expected. However `@return` line produces no error.

**Comment by romani** [comment]
Very strange how we missed that simple case. Even in documentation we claim that Check should validate returns.

**Comment by romani** [comment]
Fix is merged
